modules/ztp-configuring-a-static-ip.adoc

```diff
@@ -35,34 +35,33 @@ spec:
      - name: eth0
        type: ethernet
        state: up
-       mac-address: <mac-address> <1>
        ipv4:
          enabled: true
          address:
-           - ip: <ip-address> <2>
-             prefix-length: <public-network-prefix> <3>
+           - ip: <ip-address> <1>
+             prefix-length: <public-network-prefix> <2>
          dhcp: false
    dns-resolver:
      config:
        server:
-         - <dns-resolver> <4>
+         - <dns-resolver> <3>
    routes:
      config:
        - destination: 0.0.0.0/0
-         next-hop-address: <gateway> <5>
+         next-hop-address: <gateway> <4>
          next-hop-interface: eth0
          table-id: 254
  interfaces:
-   - name: "eth0" <6>
-     macAddress: <mac-address> <7>
+   - name: "eth0" <5>
+     macAddress: <mac-address> <6>
 ----
-<1> `mac-address` is the MAC address of the target bare metal machine, that is, the same MAC address used in the `BareMetalHost` resource.
-<2> `ip-address` is the static IP address of the target bare metal machine.
-<3> `public-network-prefix` is the static IP address’s subnet for the target bare metal machine.
-<4> `dns-resolver` is the DNS server for the target bare metal machine.
-<5> `gateway` is the gateway for the target bare metal machine.
-<6> `name` must match the name specified in the `interfaces` section.
-<7> `mac-address` must match the MAC address specified in the `interfaces` section.
+<1> `ip-address` is the static IP address of the target bare metal machine.
+<2> `public-network-prefix` is the static IP address’s subnet for the target bare metal machine.
+<3> `dns-resolver` is the DNS server for the target bare metal machine.
+<4> `gateway` is the gateway for the target bare metal machine.
+<5> `name` must match the name specified in the `interfaces` section.
+<6> `mac-address` must match the MAC address specified in the `BareMetalHost` resource.
```

Comment:
I agree with @achuzhoy comment here. I think, if anything, it should be the other way around. The mac address in the BMH CR should match *one* of the server's mac addresses, likely present in this nmstate config.

Another way to phrase this could be that *One* of the mac addresses in the nmstate config *must* match the mac address in the BMH CR. This is a 1:1 relationship.

Reply:
Good question: It is quite common to have more than one network interface in a server but, in order to simplify the docs, I would say we can keep a single interface.

To answer your question about mac addresses: The network interfaces will have different mac addresses, As far as assisted installer is concerned, we care about matching one of these mac addresses.

Reply:
After a chat with Flavio, we agreed on the following:
* The mac address on this yaml file is tied to the static IP address.
* You can configure as many static IP addresses (with their macs) as you want , but we don't have an example of that atm.
* One of the mac addresses on the target bare metal machine must match the mac address used in the `BareMetalHost` resource.

Reply:
Hi @flaper87  Happy new year :)   Please have a look at the changes I made after our chat and give me an approval or lgtm. Thank you!